src/pages/documentation/release-notes/index.md

```diff
@@ -4,6 +4,13 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## July 21, 2023
+
+### Android Assurance 2.1.1
+
+- Added support for chained events.
+- Improvements to the test app.
```

Comment:
We usually don't include non-sdk related notes in this repo, but we do in github tags. Is this a change requested by one of our customers or is there a strong reason for including it?

Reply:
I wasn't aware of that non-sdk changes can be omitted. I will remove them 👍🏼